packages/bpk-component-card-list/src/BpkCardListRowRail/BpkCardListCarousel.module.scss

```diff
@@ -17,27 +17,33 @@
  */
 
 @use '../../../unstable__bpk-mixins/tokens';
+@use '../../../unstable__bpk-mixins/breakpoints';
+@use '../../../unstable__bpk-mixins/utils';
 
 .bpk-card-list-row-rail {
   &__row,
   &__rail {
     display: flex;
     padding-top: tokens.bpk-spacing-sm();
     padding-bottom: tokens.bpk-spacing-sm();
-    overflow-x: scroll;
+    overflow-x: hidden;
     box-sizing: border-box;
     gap: tokens.bpk-spacing-sm();
     scroll-snap-stop: always;
     scroll-snap-type: x mandatory;
     scrollbar-width: none;
 
+    @include breakpoints.bpk-breakpoint-mobile {
+      overflow-x: scroll;
+    }
+
     &::-webkit-scrollbar {
       display: none;
     }
 
     &__card {
       position: relative;
-      padding: tokens.bpk-spacing-md();
+      padding: 0 tokens.bpk-spacing-md();
```

Comment:
It's parent element already set padding top/bottom, if add padding top/bottom here, will cause spacing much big than design.
<img width="2205" height="1012" alt="image" src="https://github.com/user-attachments/assets/f542346b-2d87-476f-bb6d-3a9a15ceec04" />